### PR TITLE
Scrollable content for swipeable modal

### PR DESCRIPTION
**Pull request by pistonsky**
Can't fit all content into one screen? Make it scrollable. Just like every other app out there does.

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "react": "16.0.0-alpha.6",
     "react-native": "0.43.3",
-    "react-native-modal": "3.1.0"
+    "react-native-modal": "https://github.com/pistonsky/react-native-modal.git"
```
Comment (mmazzarolo) [inline example/package.json]: Could you revert this? Thanks!

File: README.md [hunk 2]
```diff
@@ -132,6 +133,9 @@ For a more complex example take a look at the `/example` directory.
 | onModalHide                    | func             | () => null     | Called when the modal is completely hidden                                                   |
 | onModalShow                    | func             | () => null     | Called when the modal is completely visible                                                  |
 | onSwipe                        | func             | null           | Called when the `swipeThreshold` has been reached                                            |
+| scrollOffset                   | number           | 0              | When > 0, disables swipe-to-close, in order to implement scrollable content                  |
+| scrollOffsetMax                | number           | 0              | Used to implement overscroll feel when content is scrollable. See `/example` directory       |
+| scrollTo                       | func             | null           | Used to implement scrollable modal. See `/example` directory for reference on how to use it  |
```
Comment (mmazzarolo) [inline README.md]: Thanks you for updating the README.md 🙏

File: README.md [hunk 3]
```diff
@@ -13,13 +13,14 @@ An enhanced, animated and customizable react-native modal.
...
 * Swipeable
+* Scrollable
 
 ## Demo
 
 <p align="center">
 <img src="https://raw.githubusercontent.com/mmazzarolo/react-native-tips/master/imgs/modal.gif" height="300" />
 <img src="https://raw.githubusercontent.com/mmazzarolo/react-native-modal-datetime-picker/master/extras/datetimepicker-ios.gif" height="300" />
-<img src="https://raw.githubusercontent.com/mmazzarolo/react-native-animated-modal/master/extras/example-modal.gif" height="300" />
+<img src="https://raw.githubusercontent.com/pistonsky/react-native-modal/master/extras/example-modal.gif" height="300" />
```
Comment (mmazzarolo) [inline README.md]: Nice!
Do you also have any production app that uses react-native-modal?
I'll happily add a gif of it and a link to the store.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-modal",
-  "version": "5.4.0",
+  "version": "5.5.0",
```
Comment (mmazzarolo) [inline package.json]: Could you revert this? Thanks!

File: README.md (hunk 3 above)
Comment (pistonsky) [reply README.md]: Not yet. My team is getting ready to use it, and so I decided to submit this PR beforehand.

**Comment by pistonsky** [comment]
@mmazzarolo Done! But why not to increase the version number?

**Comment by mmazzarolo** [comment]
Because I’ll do it once I’ll release the new version (tomorrow morning).

**Comment by mmazzarolo** [comment]
Thanks @pistonsky :heart:

**Comment by mmazzarolo** [comment]
Landed on [v6.0.0](https://github.com/react-native-community/react-native-modal/releases/tag/v6.0.0) 🔆

**Pull request merged**